Gemfile

```diff
@@ -28,7 +28,7 @@
 
 source 'https://rubygems.org'
 
-ruby '~> 2.4.2'
+ruby '~> 2.5.0'
```

Comment:
Do we have to force Ruby 2.5.0? We could change the directive to `ruby '>= 2.4'` ?